### PR TITLE
updated readme with latest changes

### PR DESCRIPTION
**Pull request by kevinpapst**
forgot that in the last PRs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,6 +134,19 @@ And on my Windows 10 machine:
 └──────────────────────────┘
 ```
 
+macOS:
```
Comment (Pierstoval) [inline README.md:137]: Could be interesting to show what version of OSX you have, WDYT?

Comment (kevinpapst) [reply README.md:137]: I had `macOS 10.15.6` first, then changed it to `macOS Catalina` and finally I thought: "Well, it is just an example and Mac users will see that it is a Homebrew environment - the version doesn't matter". But I can add it for sure I you prefer that.

Comment (Pierstoval) [reply README.md]: Newcomers might not even be aware of Homebrew, so... I think all information are useful 😉

Comment (kevinpapst) [reply README.md]: I like your positive attitude! I couldn't imagine that a newcomer would stumble upon this tool and understands its usage before finding Homebrew 😁 but why not?! I added the version in the last commit

Comment (Pierstoval) [reply README.md]: It's more than if this project grows and it becomes a good tool for newcomers, I wish they find it newcomer-friendly 😄

File: README.md [hunk 2]
```diff
@@ -129,6 +134,19 @@ And on my Windows 10 machine:
 └──────────────────────────┘
 ```
 
+macOS Catalina:
```
Comment (Pierstoval) [inline README.md:137]:
```suggestion
macOS Catalina (using Homebrew): 
```

**Review by Pierstoval — CHANGES_REQUESTED** [review]

**Review by Pierstoval — APPROVED** [review]

**Pull request merged**